src/main/java/dev/plex/extras/command/OrbitCommand.java

```diff
@@ -0,0 +1,83 @@
+package dev.plex.extras.command;
+
+import com.google.common.collect.ImmutableList;
+import dev.plex.command.PlexCommand;
+import dev.plex.command.annotation.CommandParameters;
+import dev.plex.command.annotation.CommandPermissions;
+import dev.plex.util.PlexUtils;
+import net.kyori.adventure.text.Component;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+import org.bukkit.potion.PotionEffect;
+import org.bukkit.potion.PotionEffectType;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+@CommandParameters(name = "orbit", description = "Accelerates the player at a super fast rate", usage = "/<command> <target> [<<power> | stop>]")
+@CommandPermissions(permission = "plex.tfmextras.orbit")
+public class OrbitCommand extends PlexCommand
+{
+
+    private static final Map<UUID, Boolean> isOrbitedMap = new HashMap<>();
```

Comment:
Use list instead of map - according to the code, players in the list are orbited... No need for boolean